python/src/mas/cli/install/argParser.py

```diff
@@ -75,7 +75,15 @@ def isValidFile(parser, arg) -> str:
     required=False,
     help="Subscription channel for the Core Platform"
 )
-
+# MAS Special characters 
+# -----------------------------------------------------------------------------
+masSpecialCharacters = installArgParser.add_argument_group("Mas Special Characters")
+masSpecialCharacters.add_argument(
+    "--allow-special-chars",
+    dest="mas_special_characters",
+    required=False,
+    help="Allow special chars for users username/ID",
+)
```

Comment:
This should be a boolean ... e.g. using `action=store_true`, see `--eck-enable-logstash` for example.